log/logger.go

```diff
@@ -11,12 +11,21 @@ import (
 )
 
 // Logger holds a logger to log to.
-type Logger = log.Logger
+type Logger struct {
```

Comment:
Please revert that. We want to alias that type. not embed it. `type Logger = log.Logger` is fine.